src/components/UserLabel/UserLabel.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+
+import {Envelope, Xmark} from '@gravity-ui/icons';
+
+import {Avatar} from '../Avatar';
+import type {AvatarProps} from '../Avatar';
+import {Icon} from '../Icon';
+import {block} from '../utils/cn';
+
+import i18n from './i18n';
+import type {UserLabelProps} from './types';
+import {extractTextValue, extractTextView} from './utils';
+
+import './UserLabel.scss';
+
+const COMMON_AVATAR_PROPS: Partial<AvatarProps> = {
+    size: 's',
+};
+
+const b = block('user-label');
+
+export const UserLabel = ({
+    type = 'person',
+    avatar,
+    text,
+    size = 's',
+    view = 'outlined',
+    onClick,
+    onClose,
+    className,
+    style,
+    qa,
+}: UserLabelProps) => {
+    const textValue = extractTextValue(text);
+    const textView = extractTextView(text);
+
+    const closeButtonAriaAttributes: React.AriaAttributes = {
+        'aria-label': i18n('label_remove-button', {text: textValue}),
+    };
+
+    const clickable = Boolean(onClick);
+    const closeable = Boolean(onClose);
+    const MainComponent = clickable ? 'button' : 'div';
```

Comment:
`'button'` should be with `type="button"` attribute in order to work correctly inside forms